gtsam/geometry/Cal3Bundler.h

```diff
@@ -28,23 +28,25 @@ namespace gtsam {
  * @addtogroup geometry
  * \nosubgrouping
  */
-class GTSAM_EXPORT Cal3Bundler {
+class GTSAM_EXPORT Cal3Bundler : public Cal3 {
 
-private:
-  double f_; ///< focal length
-  double k1_, k2_; ///< radial distortion
-  double u0_, v0_; ///< image center, not a parameter to be optimized but a constant
-  double tol_; ///< tolerance value when calibrating
+ private:
+  double f_ = 1.0f;               ///< focal length
```

Comment:
why do we still have this?

Reply:
Cal3Bundler doesn't use `fx_` or `fy_` like the base class, just a single, common `f` value. Though now that I think about it, I can do something about it.